File: MedicineReminder-master/app/src/main/java/com/example/medicineremindernew/AlarmReceiverNotify.java
package com.example.medicineremindernew;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.os.Bundle;

public class AlarmReceiverNotify extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        Bundle extras = intent.getExtras();
        String name = extras.getString("name");
        String time = extras.getString("time");
        NotificationMaker.makeNotify(context, name, time);

    }
}

File: app/src/main/java/com/example/medicineremindernew/alarm/AlarmReceiverNotify.kt
package com.example.medicineremindernew.alarm

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class AlarmReceiverNotify : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        val name = intent.getStringExtra("name")!!
        val time = intent.getStringExtra("time")!!
        NotificationService.makeNotify(context, name, time)
    }
}
